### PR TITLE
Expand Data Grid to support 4.18

### PR DESCRIPTION
**Pull request by madunn**
Expand the Data Grid scenario to support OCP 4.18 with and without FIPS

### CODE REVIEW

File: ci-operator/config/infinispan/infinispan-operator/infinispan-infinispan-operator-stable__data-grid-ocp4.18-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: data-grid-interop-aws
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/infinispan/infinispan-operator/infinispan-infinispan-operator-stable__data-grid-ocp4.18-lp-interop.yaml:28]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/config/infinispan/infinispan-operator/infinispan-infinispan-operator-stable__data-grid-ocp4.18-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        ]
+      USER_TAGS: |
+        scenario data-grid
+    test:
+    - ref: data-grid-prepare-cluster
+    - ref: install-operators
+    - ref: data-grid-tests-integration
+    workflow: firewatch-ipi-aws
+- as: data-grid-interop-aws-fips
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/infinispan/infinispan-operator/infinispan-infinispan-operator-stable__data-grid-ocp4.18-lp-interop.yaml:60]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/infinispan/infinispan-operator/infinispan-infinispan-operator-stable-periodics.yaml [hunk 3]
```diff
@@ -536,3 +547,187 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/infinispan/infinispan-operator/infinispan-infinispan-operator-stable-periodics.yaml:552]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/infinispan/infinispan-operator/infinispan-infinispan-operator-stable-periodics.yaml [hunk 4]
```diff
@@ -536,3 +547,187 @@ periodics:
...
+        secretName: manifest-tool-local-pusher
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/infinispan/infinispan-operator/infinispan-infinispan-operator-stable-periodics.yaml:644]:
```suggestion
  cron: 0 0 11 6 *
```

**Comment by madunn** [comment]
Closing as duplicate of https://github.com/openshift/release/pull/57590

**Pull request closed without merging**